Tests/ApolloCodegenTests/CodeGenIR/IRRootFieldBuilderTests.swift

```diff
@@ -40,7 +40,7 @@ class IRRootFieldBuilderTests: XCTestCase {
   // MARK: - Helpers
 
   func buildSubjectRootField(operationName: String? = nil) async throws {
-    addDeferDirective()
+    schemaSDL.appendDeferDirective()
```

Comment:
We can either do this centrally, as it is now, for all localized test schemas OR be explicit about which schemas we want to support `@defer`. I'm open to the latter if you think it'd be a cleaner way to do this.

Reply:
I thought about this more last night and have decided to remove it from the common `buildSubjectRootField` function and instead use it at each schema that needs the defer directive added. My reasoning was that there are more tests that don't need it and using it on each test is more declarative of what's needed. Plus it's going to be easy to forget about this little addition if it's hidden away in a common function but at least now it's more visible and I'm hoping we'll take note of it more once the directive is available in graphql-js like all other directives.